modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/MapTransformer.scala

```diff
@@ -91,7 +91,7 @@ object MapTransformer {
   )(
     implicit m: Manifest[T]
   ): ValidatedNel[FailureDetails.EnrichmentFailure, T] = {
-    val newInst = m.runtimeClass.newInstance()
```

Comment:
I'd restrain from making changes in SCE in this very SE-specific ticket. Would be much better if it had its dedicated commit: https://github.com/snowplow/enrich/issues/252

Reply:
Sure. It was failing my compilation due to this error: 
```
[error] /enrich/modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/MapTransformer.scala:94:34: method newInstance in class Class is deprecated: see corresponding Javadoc for more information.
[error]     val newInst = m.runtimeClass.newInstance()
```

Reply:
No, it can be same PR. Just separate commit.

Just in case - it fails because you use JDK 9 or higher. It compiles just fine on JDK 8.

Reply:
It was very painful to pull out the JDK 13/14 from my system. back to work now. I will submit the fix for that one too.